pyp2rpm/archive.py

```diff
@@ -139,6 +157,21 @@ def get_content_of_file(self, name, full_path=False):  # TODO: log if file can't
 
         return None
 
+    def extract_file(self, name, full_path=False, directory="."):
+        """Extract a member from the archive to the specified working directory.
+        Behaviour of name and pull_path is the same as in function get_content_of_file.
+        """
+        if self.handle:
+            for member in self.handle.getmembers():
+                if (full_path and member.name == name)\
```

Comment:
See http://stackoverflow.com/questions/181530/python-style-multiple-line-conditions-in-ifs